### ISSUE TITLE
New template for *-analysis-steps

### ISSUE THREAD

**Issue by alexpearce** [lhcb/starterkit#23]
The current template for the lessons is based on the [software carpentry template](https://github.com/swcarpentry/lesson-example). It has served us well, but I think we can do better.

I'd like us to discuss two things: moving to a different lesson template, and the possibility of merging the two lesson repos.
### Pros
1. It works! We've been using the lesson templates for three workshops now.
2. It doesn't look _terrible_.
3. We've had to spend very little time fiddling with layouts and CSS and `blah`, just concentrated on the content.
### Cons
1. We never got around to fully customising the template; e.g. some links in the footer still point to Software Carpentry stuff.
2. It doesn't look _particularly_ nice. Could have proper syntax highlighting, could match the style of the [central site](https://lhcb.github.io/starterkit), have a better layout, and so on.
3. We would have to do some designing/website building, and it's hard enough getting the content ready in time.
## Having a single lessons repository

This is just an idea I had.

I'm not sure if there's a big benefit in having two repos, first- and second-analysis steps. We could just have a single `lhcb-starterkit-lessons` repo, which we can pick and chose from when putting together a course. Then we don't have to have clumsy links referring to lessons in “the first Starterkit” or “in First Analysis Steps”; everything will be easy to find because it's all on one page.

(I had students in the Impactkit who didn't realise/forgot that we'd already taught some things in first-analysis-steps that they could use for stuff in second-*. Having all the lessons visible on a single index might help.)

**Comment by kdungs** [comment]
I agree with everything you said. Don't forget we also have `analysis-essentials` which is based on `gitbook`.

**Comment by apuignav** [comment]
Agree too, we need to cleanup our mess :-)

On Sun, May 22, 2016 at 5:40 PM, Kevin Dungs notifications@github.com
wrote:

> I agree with everything you said. Don't forget we also have
> analysis-essentials which is based on gitbook.
> 
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> https://github.com/lhcb/starterkit/issues/23#issuecomment-220838960

## 

Dr. Albert Puig Navarro
Laboratoire de Physique des Hautes Energies
Ecole Polytechnique Fédérale de Lausanne (EPFL)
BSP 614.4 (Cubotron UNIL) CH-1015 Lausanne
EPFL Phone: 021 6939808
CERN Phone: 72518

**Comment by ibab** [comment]
I'm in favour of moving everything into a single gitbook-based repo.
(Or a custom web page, if someone can find the time to design it)
But we should keep in mind that our current content has already been indexed extremely well, so there will be a painful transition period where people won't be able to find the new content.

Maybe it would be best to keep the current lessons as they are, and start building a `starterkit-ng` repo in parallel?

**Comment by alexpearce** [comment]
I hadn't thought of indexing, it's a good point. We could keep the old sites up and add a "go here" link. Or we could add a `meta` tag to redirect to the new site. I think Google changes search results to the new page if it finds a redirect (I'm not sure though).

**Comment by apuignav** [comment]
If only we knew somebody working at the big G to clear this up...

**Comment by betatim** [comment]
Check out https://github.com/jekyll/jekyll-redirect-from#redirect-to for redirecting to new pages.

Considering the lhcb org is mainly used by the kits I'd keep using it. Until the day when the starterkit goes LHC wide ;)

**Comment by alexpearce** [comment]
@betatim That's only for pages built with [Jekyll](https://jekyllrb.com/) I think.
